cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java

```diff
@@ -392,11 +410,69 @@ private void addModelDescriptionToCgmes() {
         }
     }
 
-    private static String fs(double value) {
+    private static PropertyBag buildSvVoltage(String tpNode, double voltage, double angle) {
+        PropertyBag p = new PropertyBag(SV_VOLTAGE_PROPERTIES);
+        p.put(CgmesNames.TOPOLOGICAL_NODE, tpNode);
+        p.put(CgmesNames.VOLTAGE, fstr(voltage));
+        p.put(CgmesNames.ANGLE, fstr(angle));
+        return p;
+    }
+
+    private PropertyBag buildSvPowerFlow(Terminal terminal, int sequenceNumber) {
+        // TODO If we could store a terminal identifier in IIDM
```

Comment:
And about terminals at  EquivalentInjection at boundaries and terminals of connected equipments at boundary TN ? Do you support them here ?